Src/main.c

```diff
@@ -133,7 +133,7 @@ int main(void)
       /*Total period is 20 ms*/
       /*Changing the pulse width (between 1ms to 2ms) will change the angle of the shaft of the "servo motor"*/
       /*change from 1.5 ms to 1ms will rotate from 0 degrees to -90 degrees. 1.5ms to 2 ms will rotate 0 degrees to 90 degrees*/
-      float percentage = (raw_data/pow(2, ADC_RESOLUTION)); //percentage = ADC reading/Resolution of ADC - Resolution od ADC = 2^(# of bits)
+      float percentage = (raw_data/(pow(2, ADC_RESOLUTION)-1)); //percentage = ADC reading/Resolution of ADC - Resolution od ADC = 2^(# of bits) - ADC_RESOLUTION = 12
```

Comment:
Sorry for not mentioning this earlier. It slipped my mind

Another thing to note: 

The `pow()` function is actually incredibly inefficient. You would be better off writing `4095.0` since we are using a constant.